src/app/components/OfflineBanner/hook.ts

```diff
@@ -24,6 +24,7 @@ export const useIsApiReachable = (
 export type FreshnessInfo = {
   unavailable?: boolean
   outOfDate?: boolean
```

Comment:
It looks like redundancy in having two vars for handling one state. Doesn't `outOfDate` duplicate outOfDateReason? Would be simpler to have single source of truth for this state checking.

Reply:
Yes
That can be a separate refactoring, I think it would require some hundred lines of changes in existing code and I'd rather avoid conflicts